Code/Editor/EditorFramework/Assets/SimpleAssetDocument.h

```diff
@@ -108,16 +108,22 @@ class ezSimpleAssetDocument : public BaseClass
     : BaseClass(szDocumentPath, EZ_DEFAULT_NEW(ezSimpleDocumentObjectManager<PropertyType>), engineConnectionType)
     , m_LightSettings(bEnableDefaultLighting)
   {
-    ezEditorPreferencesUser* pPreferences = ezPreferences::QueryPreferences<ezEditorPreferencesUser>();
-    pPreferences->ApplyDefaultValues(m_LightSettings);
+    if (bEnableDefaultLighting)

```

Comment:
@Sanakan8472 if bEnableDefaultLighting was false the lighting was still enabled via the default settings, so I added this check here. I guess that was your intention with the bEnableDefaultLighting bool?

Reply:
Yes that was wrong. It probably only worked as long as I didn't have the editor preferences fully implemented.